### ISSUE TITLE
Feature requests for DTR

### ISSUE THREAD

**Issue by degenorate** [drtshock/Factions#242]
The Death Til Raidable is becoming highly popular game mode for hardcore factions. No longer is your power linked to how much land you can claim. The idea is pretty simple. 
- A faction has max chunks they claim, etc. 50 chunks of land, that has nothing to do with  /f power. 
- Each player contributes a total of 0.51 DTR to a total faction DTR of 5.5, no matter how many faction members. 
- When a player joins a faction their DTR starts adding .01 DTR every 12 seconds up to a total of 0.51.
- When a player dies the faction loses 1.01 DTR.
  - No land in the faction is raidable until the factions DTR hits 0, all land becomes raidable for a period of time (configurable)
- Other Factions can only raid, not overclaim. 
- /dtr to display current Death Til Raidable Ratio of faction instead of /f power
- /f who, would show the /f home coords for any faction, if set.

**Comment by drtshock** [comment]
Such wall of text. tl;dr please.

**Comment by degenorate** [comment]
Edited to make it hopefully more readable.

**Comment by degenorate** [comment]
Have you got a chance to look at this yet?

**Comment by oFrostMC** [comment]
Its normally 36 chunks of land not 50, just reminding you.

**Comment by degenorate** [comment]
Yeah that's fine, I was hoping it was or could be configurable. But if it's set to 36 that would be perfectly fine. I just wanted to explain more because up to now I think drtshock believes that you could achieve death til raidable just by changing player and death power ratio in old factions config. Whi chi simply isn't the case, cuz it's a completely different way of game play really.

**Comment by nscwalk** [comment]
You can bypass this by setting everyone's default power to 55 and on death you lose 100 power. Then set the max power to 550. Then make the maximum land claim 50 or 36 whatever you want. All it will do is get rid of a decimal so instead of seeing 5.50 DTR you see 550 easy solution.

**Comment by degenorate** [comment]
No its not, the power system in place in factions still allows your land to be raidable when power declines. If you lose 100 power, 100 chunks of land would be raidable, also it would allow you to claim that huge amount of land per faction. This isn't how DTR works.

**Comment by nscwalk** [comment]
then you disable land overclaim in the config........ and you can set the maximum amount of land you can also claim in the config. I dont see your problem here. my problem is I cant get it to display dtr in the first place.

**Comment by degenorate** [comment]
This still doesn't fix dtr, it's completely in the early factions raid another faction. Please stop commenting if you don't understand.  You are confusing drtshock and the reason I made this thread in the first place.

**Comment by nscwalk** [comment]
I know how hardcore factions work. everyone gets .51 DTR and a faction has a maximum of 36 land claim. and your land isnt determained by power or anything. and each time a player dies they lose 1 dtr. Im trying to give you a way you can bassicly use the plugin by editing everything in the config. and how am I confusing DRTShock hes a developer he probaly know what your trying to say and what im trying to explain to you. and to be honest if you looked at the other threads you would see the same thing. plus im trying to help you because Trent isnt going to help you as you can see he hasnt commented one time in the past 5 days.

**Comment by drtshock** [comment]
DTR will be added in a separate project. I'll post all the details on spigot and twitter soon.
